src/Widgets/DataFormNumberWidget/DataFormNumberWidgetComponent.tsx

```diff
@@ -56,18 +73,39 @@ provideComponent(DataFormNumberWidget, ({ widget }) => {
         </>
       ) : null}
       <br />
-      <input
-        className="form-control"
-        defaultValue={defaultValue}
-        id={id}
-        name={attributeName ?? ''}
-        placeholder={widget.get('placeholder')}
-        required={widget.get('required')}
-        type="number"
-        min={widget.get('minValue') ?? undefined}
-        max={widget.get('maxValue') ?? undefined}
-        step={widget.get('stepValue') ?? undefined}
-      />
+      <div className="input-group flex-nowrap">
+        <button
+          aria-label="-"
+          className="btn btn-primary btn-sm"
+          disabled={hasValue && min !== undefined && defaultValue <= min}
+          onClick={down}
+        >
+          <i className="bi bi-dash-lg px-0 text-white" />
+        </button>
+        <input
+          className="form-control text-center no-arrows"
```

Comment:
The styling currently still has some smaller issue:

1. If you click into the input box, the box becomes a bit wider and makes the +/- button a bit smaller:

https://github.com/user-attachments/assets/4e08d589-4fd7-429a-9f4d-5879b6bc4cf9

2. If you click into the input box, a drop-shadow is rendered - but only around the box. I think it would be good, if the drop shadow covers also the +/- buttons as well.
3. The input box looks like being one or two pixels smaller as the surrounding +/- buttons. I think they should have the same height.
  <img width="165" alt="Screenshot 2025-03-07 at 09 04 44" src="https://github.com/user-attachments/assets/9f8a730a-25d2-478d-ba99-70729cb0dc6f" />


If I'm not mistaken this applies to both the data number widget and the product component.

Reply:
1 and 2 have been fixed in 8731301.
3 is an optical illusion caused by the grey border, which makes the input field look like it's embossed while the buttons aren't. I could've removed the border in favor of all-white, but I'd rather stick with the current default and leave further polishing to a designer.